📝 Add clarity to metrics (#2668)
platformsh/platformsh-docs: docs/src/increase-observability/metrics/grid.md

and CPU and disk space for service containers.
These metrics are available for all of your Grid environments.

> Summaries of the metrics for each container separately can be seen in the Console under **Services** for a given environment.

Reviewer: 🛠️ 

```suggestion
You can see summaries of the metrics for each container separately in the Console under **Services** for a given environment.
```
